app/src/main/res/values/strings.xml

```diff
@@ -20,17 +20,17 @@
     <!-- Long click options -->
     <string name="long_click_title">What do you want to do?</string>
     <string name="long_click_open">Open \"%1s\"</string>
-    <string name="long_click_store">See its store page</string>
+    <string name="long_click_store">View store page</string>
     <string name="long_click_settings">Access its settings</string>
     <string name="long_click_remove_shortcut">Remove this shortcut</string>
 
     <!-- About -->
     <string name="app_name">Discreet Launcher</string>
     <string name="author" translatable="false">Vincent Falzon</string>
-    <string name="licence_name">GNU General Public License v3.0</string>
+    <string name="licence_name">GPLv3+</string>
     <string name="license_link" translatable="false">https://gnu.org/licenses/gpl-3.0.txt</string>
-    <string name="website_description">To access the source code, report an issue or see my other projects, visit the website</string>

```

Comment:
No Oxford comma, and three clauses for no good reason. It being a website is already clear from the link underneath it.

Reply:
> No Oxford comma

What is an Oxford comma? (genuine question, there is only one comma on my keyboard and I used that one)

> It being a website is already clear from the link underneath it.

Indeed, ok for me to remove that part.

---

However, the first half of the remaining sentence now tells something quite different than before...
Maybe another solution would be to simply drop this sentence and put the website link below my name.

Reply:
An Oxford comma is the same comma, but it makes whatever follows it a distinct part.
"Ride an elevator, a car and a motorbike."  (if you will) means the car and the motorbike both at the same time.
"Ride an elevator, a car, and a motorbike." is better.

"report an issue or see my other projects" should have a comma with an and/or.
The original stating it shouldn't be both is more the issue here, and "and/or" is awkward, so that is why I rewrote it.

Reply:
Thank you for the explanation about the comma, now I see what you mean.

I am still not satisfied with the new sentence and there were already flaws in the original (aside the grammar mistakes and website redundancy).

First the "other projects" was in case of future content on my website (other than Android apps). However, for now there is only Discreet Launcher and it may not even change in 2021.
=> Probably this part should be dropped.

"To access the souce code" was no more than a way to emphasize it is publicly available.
Although new features and improvement suggestions are welcomed, I prefer to manage all the coding part myself (the project is small enough to allow it and doing code reviews is not really my cup of tea). "Improve the code" goes in the opposite direction by actively asking for direct code contributions...

Here are two other ideas that I have to replace this sentence (I prefer the first one and put it as a suggestion).

**"Suggest improvements and report issues on"** (+ link)

"Source code and issues tracking on" (+ link)

Reply:
Tat is more succinct. My idea was trying to communicate you can change stuff yourself as a user.
That can be added in its own point later on.